applications/ContactStructuralMechanicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -256,9 +262,13 @@ void  AddCustomStrategiesToPython(pybind11::module& m)
     //*************************BUILDER AND SOLVER*************************
     //********************************************************************
 
-    typedef ContactResidualBasedBlockBuilderAndSolver< SparseSpaceType, LocalSpaceType, LinearSolverType > ContactResidualBasedBlockBuilderAndSolverType;
+    // Contact block builder and solver
     class_< ContactResidualBasedBlockBuilderAndSolverType, typename ContactResidualBasedBlockBuilderAndSolverType::Pointer, BuilderAndSolverType > (m, "ContactResidualBasedBlockBuilderAndSolver")
     .def(init< LinearSolverType::Pointer > ());
+
+    // Contact block buiklder and sokver with constraints
+    class_< ContactResidualBasedBlockBuilderAndSolverWithConstraintsType, typename ContactResidualBasedBlockBuilderAndSolverWithConstraintsType::Pointer, BuilderAndSolverType > (m, "ContactResidualBasedBlockBuilderAndSolverWithConstraints")
```

Comment:
I think  `typename` required here for Pointer !

Reply:
It is in the pointer

~~~
class_< ContactResidualBasedBlockBuilderAndSolverWithConstraintsType, typename ContactResidualBasedBlockBuilderAndSolverWithConstraintsType::Pointer, BuilderAndSolverType > (m, "ContactResidualBasedBlockBuilderAndSolverWithConstraints")
~~~

Reply:
Conclussion->Should I remove it?

Reply:
yes .. please